### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by Farida-Hamid**
>In this project, I added some functionality to my application to make it interactive. The user will be able to mark task completion by selecting the corresponding checkbox (or undo it by unchecking the checkbox). The updated tasks list will be stored in local storage.

## Steps:
- Add a new JavaScript file and import it as a module:
- Add event listener to the checkbox.
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button.
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/checker.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+export const clearScreen = (list) => {
+  const clear = document.querySelector('#clear');
+  clear.addEventListener('click', () => {
+    list = list.filter((item) => item.completed === false);
+    localStorage.setItem('memory', JSON.stringify([]));
+    const listContainer = document.querySelector('#list');
+    listContainer.innerHTML = '';
+  });
+};
```
Comment (phelian23) [inline src/checker.js]: - [x] The `clear completed` button should only clear items that are completed, however, at the moment it's clearing all items, kindly fix it

File: src/checker.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export const completeTask = (list) => {
+  const boxs = document.querySelectorAll('#done');
+  boxs.forEach((box, index) => {
+    box.addEventListener('click', () => {
+      list[index].completed = !list[index].completed;
+      localStorage.setItem('memory', JSON.stringify(list));
+    });
+  });
+};
```
Comment (phelian23) [inline src/checker.js]: - [x] When the checkbox is clicked, the item status changes to true, however, the item should also be struck out, and on reload, the checkbox for completed tasks should stay checked and struck out. Kindly apply this

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @Farida-Hamid,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/checker.js (hunk 2 above)
Comment (Farida-Hamid) [reply src/checker.js]: All done. Thank you.

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**